### PR TITLE
CI: fix chocolatey error

### PR DESCRIPTION
**Pull request by kroening**
Chocolatey reports "Cannot read keys when either application does not have a console".

- [X] Each commit message has a non-empty body, explaining why the change was made.
- n/a Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- n/a The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- n/a Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- n/a My commit message includes data points confirming performance improvements (if claimed).
- [X] My PR is restricted to a single feature or bugfix.
- n/a White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: .github/workflows/pull-request-checks.yaml [hunk 1]
```diff
@@ -681,6 +681,7 @@ jobs:
   # This job takes approximately 49 to 70 minutes
   check-vs-2019-cmake-build-and-test:
     runs-on: windows-2019
+    if: false
```
Comment (tautschnig) [inline .github/workflows/pull-request-checks.yaml:684]: Why is this job being disabled?

File: .github/workflows/pull-request-checks.yaml [hunk 2]
```diff
@@ -805,7 +806,7 @@ jobs:
 
   # This job takes approximately 7 to 32 minutes
   windows-msi-package:
-    runs-on: windows-2019
+    runs-on: windows-2022
```
Comment (tautschnig) [inline .github/workflows/pull-request-checks.yaml:809]: Why is this necessary/the right thing to do?

**Comment by kroening** [comment]
It works again.
